cirq-google/cirq_google/workflow/quantum_executable.py

```diff
@@ -0,0 +1,157 @@
+import abc
+import dataclasses
+from dataclasses import dataclass
+from typing import Union, Tuple, Optional, Sequence, cast
+
+import cirq
```

Comment:
Is it common practice now to just `import cirq` and then have a bunch of module and direct function imports below it. This seems pretty haphazard and far from the recommended approach of just importing modules.

Reply:
changed `from cirq` to only import modules.